src/auth/computeclient.ts

```diff
@@ -61,15 +64,18 @@ export class Compute extends OAuth2Client {
    */
   protected async refreshToken(refreshToken?: string|
                                null): Promise<GetTokenResponse> {
-    const url =
-        this.tokenUrl || `${HOST_ADDRESS}${Compute._GOOGLE_OAUTH2_TOKEN_URL}`;
+    const url = this.tokenUrl ||
+        `${gcpMetadata.HOST_ADDRESS}${Compute._GOOGLE_OAUTH2_TOKEN_URL}`;
     let res: AxiosResponse<CredentialRequest>|null = null;
     // request for new token
     try {
       // TODO: In 2.0, we should remove the ability to configure the tokenUrl,
       // and switch this over to use the gcp-metadata package instead.
-      res = await this.transporter.request<CredentialRequest>(
-          {url, headers: {'Metadata-Flavor': 'Google'}});
+      res = await ax.request<CredentialRequest>({
+        url,
+        headers: {[gcpMetadata.HEADER_NAME]: 'Google'},
```

Comment:
Could you use `headers: gcpMetadata.HEADERS` or is that not released yet?

Reply:
or rather `headers: Object.assign({}, gcpMetadata.HEADERS)`

Reply:
It's not released yet.  There's one more bug in there I'd like to land before 0.6.0.  Tracking here: https://github.com/stephenplusplus/gcp-metadata/issues/35